db/migrate/20231010153003_create_participant_id_changes.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateParticipantIdChanges < ActiveRecord::Migration[7.0]
+  def change
+    create_table :participant_id_changes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
if someone was to delete this user, what would happen 🤔

Reply:
@cwrw 
this should be fine as it will move to a new user, it should never link to a user that doesn't exist.